src/Behat/Config/Filter/Filter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Filter;
+
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Name\FullyQualified;
+
+class Filter implements FilterInterface
```

Comment:
I created a base class for all named filters and moved most of the functionality of these filters into this base class